sentinelone/komand_sentinelone/connection/connection.py

```diff
@@ -43,10 +56,15 @@ def get_auth_token(self, url, username, password):
         }
 
         r = requests.post(final_url, json=auth_headers)
+        if r.status_code == 401:
+            raise ConnectionTestException(
+                cause="Could not authorize with SentinelOne instance at: " + final_url,
+                assistance="Check if you login with e-mail. Response was: " + r.text
```

Comment:
For this assistance, are users able to login with email address or do they need something else?  Think this could be a bit more obvious.